src/framegrab/grabber.py

```diff
@@ -181,6 +206,15 @@ def grab(self) -> np.ndarray:
         """
         pass
 
+    def _autoassign_name(self) -> None:
+        """For generating and assigning unique names for unnamed FrameGrabber objects.
```

Comment:
For auto-assigning names, can we use `ksuid`, so that the name would be something like `unnamed_2T5OEK0BqNepz5zagJxQkiX6iCy` where `2T5OEK0BqNepz5zagJxQkiX6iCy` is the generated ID. This is not strictly necessary, but it's nice because it follows other conventions we have in the rest of our codebase.